### DISCUSSION TITLE
plays/context-with-real-usecases/

### DISCUSSION THREAD

**Discussion by giscus** [reactplay/react-play#361] [Play Feedback]
# plays/context-with-real-usecases/

context-with-real-usecases is a recipe app that lists the recipes by cuisine type. Based on the selected cuisine app shows the list of recipes, the selected cusine and the cusine fact.

https://reactplay.io/plays/context-with-real-usecases/

**Comment by koustov** [comment]
Awesome play @Deepak8717

**Comment by Deepak8717** [comment]
@koustov thanks

**Comment by vasantisuthar** [comment]
Its really an amazing and helpful play @Deepak8717.

**Reply by Deepak8717** [reply]
@vasantisuthar thanks

**Comment by Shivam-Katare** [comment]
Great Play @Deepak8717
